api/v1/models/invitation.py

```diff
@@ -0,0 +1,18 @@
+from sqlalchemy import Column, String, Integer, ForeignKey, Table, Boolean, DateTime, func
+from sqlalchemy.orm import relationship
+from api.v1.models.base import Base
+from api.v1.models.base import role_permission_association, user_role_association
+from uuid_extensions import uuid7
+from sqlalchemy.dialects.postgresql import UUID
+
+# Invitation model
+class Invitation(Base):
+    __tablename__ = 'invitations'
+
+    id = Column(UUID(as_uuid=True), primary_key=True, default=uuid7)
+    user_id = Column(UUID(as_uuid=True), ForeignKey('users.id', ondelete='CASCADE'), nullable=False)
+    organization_id = Column(UUID(as_uuid=True), ForeignKey('organizations.id', ondelete='CASCADE'), nullable=False)
+    created_at = Column(DateTime(timezone=True), server_default=func.now())
+    updated_at = Column(DateTime(timezone=True), server_default=func.now(), onupdate=func.now())
+    expires_at = Column(DateTime(timezone=True), nullable=False)
+    is_valid = Column(Boolean, default=True)
```

Comment:
An abstract BaseModel class was created to give every model a set of base fields and functionality, inherit from that and remove duplicate fields --- Do this for all classes

Reply:
when merged 

sunday will fix it 

i dont know anything here